hack/update-codegen.sh

```diff
@@ -97,3 +97,35 @@ do
     paths="${APIS_PKG}/apis/${VERSION}" 
 
 done
+
+echo "Generating gRPC/Protobuf code"
+
+readonly PROTOC_CACHE_DIR="/tmp/protoc.cache"
+readonly PROTOC_BINARY="${PROTOC_CACHE_DIR}/bin/protoc"
+readonly PROTOC_URL="https://github.com/protocolbuffers/protobuf/releases/download/v22.2/protoc-22.2-linux-x86_64.zip"
```

Comment:
abstract out version, arch, os using some VAR ?

Reply:
I've added support for arm64 Linux here, but I wasn't aware that this repo supported OSs besides Linux. If so, I might have to do something about the number of checksums in the mix here.

Reply:
I use `darwin`, would assume many contributors are using it as well